tests/Makefile

```diff
@@ -1,132 +1,239 @@
--include .local.make
-DOCKER_IMAGE_NAME := docker.io/zenko/zenko-e2e
-DOCKER_IMAGE_TAG := latest
-
-DOCKER := docker
-TOX := tox
+# Some bin aliases
+DOCKER := @docker
+TOX := @tox
+HELM := @helm
```

Comment:
Maybe it's not a good idea to suppress all echoing by default with `@`s. Since most stuff is being suppressed anyway, why not just run by default with the `-s` or `--silent` flag, which is easier to override or change.

Reply:
we could also use the same format as metalk8s's Makefile with a `$(V)` variable being defined, to toggle verboseness

Reply:
Yep, or something like:
```bash
ifndef VERBOSE
MAKEFLAGS	+= --no-print-directory --silent
endif
```